llvm/test/Transforms/LoopInterchange/interchange-no-deps.ll

```diff
@@ -1,22 +1,42 @@
-; REQUIRES: asserts
-; RUN: opt < %s -passes='loop(loop-interchange),simplifycfg' -cache-line-size=64 -simplifycfg-require-and-preserve-domtree=1 -pass-remarks-output=%t \
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt < %s -passes='loop(loop-interchange),simplifycfg' -simplifycfg-require-and-preserve-domtree=1 \
```

Comment:
Why does this drop the check lines for the remarks? Those don't require asserts, only `-stats` does I think, which is still here?

Reply:
Also, why drop `-cache-line-size=64`?